docs/source/configuration/telemetry.md

```diff
@@ -31,38 +31,32 @@ which will prompt you for your consent the first time.
 For technical information on how the telemetry collection works, you can browse
 [the source code of `kedro-telemetry`](https://github.com/kedro-org/kedro-plugins/tree/main/kedro-telemetry).
 
-## How do I consent to the use of Kedro-Telemetry?
-
-Kedro-Telemetry is a Python plugin. To install it:
-
-```console
-pip install kedro-telemetry
-```
-
-```{note}
-If you are using an official [Kedro project template](/starters/starters) then `kedro-telemetry` is included in the [project-level `requirements.txt`](/kedro_project_setup/dependencies) of the starter. `kedro-telemetry` is activated after you have a created a new project with a [Kedro project template](/starters/starters) and have run `kedro install` from the terminal.
-```
+## How do I withdraw consent?
 
-When you next run the Kedro CLI you will be asked for consent to share usage analytics data for the purposes explained in the privacy notice, and a `.telemetry` YAML file will be created in the project root directory. The variable `consent` will be set according to your choice in the file, e.g. if you consent:
+To withdraw consent, you have a few options:
```

Comment:
Shall we also add some explanation on what happens if several options are applied at the same time, and how they redefine each other?

Relates to: https://github.com/kedro-org/kedro-plugins/issues/785

Reply:
I think we should treat https://github.com/kedro-org/kedro-plugins/issues/785 as an unfortunate exception... Like, adding a warning in the docs that says

> For now, if your `.telemetry` file contains `consent: true`, it will take precedence over the rest of the methods. To avoid it, remove your `.telemetry` file.

And probably, in the future we should de-emphasize the `.telemetry` file method to avoid this sort of thing.

Reply:
@astrojuanlu, I believe the proposed warning text is not accurate. Currently, environment variables take precedence, except in rare cases described in [this issue](https://github.com/kedro-org/kedro-plugins/issues/785). Are you suggesting that we include a warning about those specific cases? We could write: "In some rare cases, the `.telemetry` file will take precedence..."